code/modules/mob/living/basic/farm_animals/bee/_bee.dm

```diff
@@ -258,6 +258,9 @@
 	var/datum/reagent/chemical = needle.reagents.get_master_reagent()
 	if(isnull(chemical))
 		return
+	if(!(chemical.chemical_flags & REAGENT_CAN_BE_SYNTHESIZED))
+		to_chat(user, span_warning("[queen] does not react to the contents of the syringe. It appears that the contents are not able to be synthesized naturally!"))
+		return
```

Comment:
the text is a bit weird considering you dont actually inject the syringe yet.

Should be something like 

` "[chemical.name] cannot be inserted into a bee's genome."`

Reply:
hash this out with @ZephyrTFA

Reply:
shit talking my code recommendation, smh smh

Reply:
```suggestion
	if(!(chemical.chemical_flags & REAGENT_CAN_BE_SYNTHESIZED))
		to_chat(user, span_warning("[chemical.name] cannot be inserted into a bee's genome."))
		return
```